{ 'log_level':'ERROR    ', 'time_stamp':'02/04/2016 02:19:28 PM CST', 'logger_name':'__main__    ', 'path_to_file':'./smart.py', 'function_name':'process_arguments', 'line_number':'207', 'message':'command failed' }
{ 'log_level':'CRITICAL ', 'time_stamp':'02/04/2016 02:19:28 PM CST', 'logger_name':'__main__    ', 'path_to_file':'./smart.py', 'function_name':'process_arguments', 'line_number':'210', 'message':'' }
{ 'log_level':'ERROR    ', 'time_stamp':'02/04/2016 02:19:40 PM CST', 'logger_name':'__main__    ', 'path_to_file':'./smart.py', 'function_name':'process_arguments', 'line_number':'207', 'message':'command failed' }
{ 'log_level':'CRITICAL ', 'time_stamp':'02/04/2016 02:19:40 PM CST', 'logger_name':'__main__    ', 'path_to_file':'./smart.py', 'function_name':'process_arguments', 'line_number':'210', 'message':'' }
{ 'log_level':'CRITICAL ', 'time_stamp':'03/30/2016 07:49:04 PM CDT', 'logger_name':'__main__    ', 'path_to_file':'smart.py', 'function_name':'process_arguments', 'line_number':'210', 'message':'global name 'surpass' is not defined' }
